nixos/lib/test-driver/test-driver.py

```diff
@@ -739,11 +742,12 @@ def create_socket(path: str) -> socket.socket:
         self.shell, _ = self.shell_socket.accept()
 
         def process_serial_output() -> None:
-            for _line in self.process.stdout:
-                # Ignore undecodable bytes that may occur in boot menus
-                line = _line.decode(errors="ignore").replace("\r", "").rstrip()
-                eprint("{} # {}".format(self.name, line))
-                self.logger.enqueue({"msg": line, "machine": self.name})
+            if self.process.stdout:
```

Comment:
same here. if this is `None`, we are in trouble, so asserting this would be fine.

Reply:
will do.